src/js/index.js

```diff
@@ -0,0 +1,130 @@
+}
+
+function hide(...els) {
+  var el, i, len
+  const results = []
+  for (i = 0, len = els.length; i < len; i++) {
+    el = els[i]
+    if (el) {
+      results.push((el.style.display = 'none'))
+    }
+  }
+  return results
+}
+
+function setMinVersion(version = 11) {
+  version = parseInt(version)
+  const ref = document.querySelectorAll('.comparison')
+  const results = []
+  for (let i = 0, len = ref.length; i < len; i++) {
+    const section = ref[i]
+    const blocks = section.querySelectorAll('.browser')
+    const versions = {}
+    for (let j = 0, len1 = blocks.length; j < len1; j++) {
+      const block = blocks[j]
+      versions[block.getAttribute('data-browser')] = block
+    }
+    switch (version) {
+      case 8:
+        showFirst(versions['ie8'])
+        results.push(hide(versions['ie9'], versions['ie10']))
+        break
+      case 9:
+        showFirst(versions['ie9'], versions['ie8'])
+        results.push(hide(versions['ie10']))
+        break
+      case 10:
+        showFirst(versions['ie10'], versions['ie9'], versions['ie8'])
+        results.push(hide(versions['ie11']))
+        break
+      case 11:
+        results.push(
+          showFirst(
+            versions['ie11'],
+            versions['ie10'],
+            versions['ie9'],
+            versions['ie8']
+          )
+        )
+        break
+      default:
+        results.push(void 0)
+    }
+  }
+  return results
+}
+
+function filter(term) {
+  let visibleIndex = 0
+  let allEmpty = true
+  const ref = document.querySelectorAll('section')
+  for (const i = 0, len = ref.length; i < len; i++) {
+    const section = ref[i]
+    let empty = true
+    const ref1 = section.querySelectorAll('.comparison')
+    for (const j = 0, len1 = ref1.length; j < len1; j++) {
+      const comp = ref1[j]
+      if (
+        !term ||
+        comp.textContent.toLowerCase().indexOf(term.toLowerCase()) !== -1
+      ) {
+        empty = false
+        comp.classList.remove('hidden')
+      } else {
+        comp.classList.add('hidden')
+      }
+    }
+    if (empty) {
+      section.classList.add('hidden')
+    } else {
+      allEmpty = false
+      section.classList.remove('hidden')
+      if (++visibleIndex % 2) {
+        section.classList.add('odd')
+      } else {
+        section.classList.remove('odd')
+      }
+    }
+  }
+  const comparisons = document.querySelector('.comparisons')
+  if (allEmpty) {
+    return comparisons.classList.add('empty')
+  } else {
+    return comparisons.classList.remove('empty')
+  }
+}
+
+document.addEventListener('DOMContentLoaded', function () {
+  const slider = document.querySelector('.version-slider')
+  const stars = document.querySelector('.github-stars')
+  fetch('https://api.github.com/repos/hubspot/youmightnotneedjquery')
+    .then((r) => r.json())
+    .then((data) => (stars.textContent = numberWithCommas(data.watchers_count)))
+    .catch(() => (stars.textContent = '10k+'))
+  function handleChange() {
```

Comment:
The tiniest of nits: the original coffeescript code generated a closure here and fired the `handleChange` function on load. This will have to be manually recreated or at least invoked once because all elements display on initial load

Reply:
Thanks, didn't realized that's what `do` was for